DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/AvatarOperations.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Constants;
+using GitHubExtention.Preferences.WebApi.Extensions;
+using GitHubExtention.Preferences.WebApi.Models;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public static class AvatarOperations
+    {
+        public static async Task<string> GetNewAvatarUrl(this IAzureContainerQuery container, string url)
```

Comment:
This method calls when user firstly register in our system and only way to get image url is to send request to GitHub get image,  save this  image to blobStorage and finally get the image url. Maybe it better to make two method one for getting image file and second for saving this image to blobStorage.